app/controllers/carts_controller.rb

```diff
@@ -54,7 +53,20 @@ def new_quantity
 
   def order_item_in_session
     item = Item.find_by_dao(params[:item_id])
-    OrderItem.where(id: session[:cart], joke_id: session[:joke_id], item_id: item.id).first
+    OrderItem.where(id: session[:cart], joke_id: session[:joke_id],
+                    item_id: item.id).first
+  end
+
+  def update_orderitem
+    OrderItem.find(params[:order_item][:order_item_id])
+                  .update(quantity: params[:order_item][:quantity])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 14) spaces for indenting an expression spanning multiple lines.